python/frontend/cirrus/examples/hyperparameter_optimization.ipynb

```diff
@@ -20,7 +20,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
Can you explain what this does?

Reply:
The collapse parameter is there, since I probably accidentally collapsed the output in jupyter notebook before committing. It doesn't change any of the code and changes the format very little. I don't think it matters.